build.js

```diff
@@ -164,12 +165,37 @@ function buildLocale (source, locale) {
 
   // This actually executes the build and stops the internal timer after
   // completion.
-  metalsmith.build((err) => {
+  metalsmith.build((err, files) => {
```

Comment:
Nit: was the `files` parameter added by mistake ? It seems useless here.

Reply:
@ZYSzys you were right. I must have added it somewhere along the way. Fixed it.

@fhemberger @ZYSzys Thank you!